app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -150,10 +158,6 @@ public void onWebGuiAvailable() {
             }
             tryIsAvailable();
         });
-        getSystemInfo(info -> {
```

Comment:
I think the tryIsAvailable should not be there in case of configSaved?

Reply:
I tried on my phone, you're right. If tryIsAvailable() is there, the app just crashes.